Lagom 1.6.1

dwijnand opened the pull request.

dwijnand committed:
Lagom 1.6.1

ihostage commented on lightbend-telemetry/lightbend-telemetry-java-mvn/pom.xml:
```diff
@@ -45,7 +45,7 @@
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>exec-maven-plugin</artifactId>
-                    <version>1.6.0</version>
+                    <version>1.6.1</version>
```
Miss :smile: It is not a Lagom version.

ihostage commented on lightbend-telemetry/log-correlation-java-mvn/pom.xml:
```diff
@@ -45,7 +45,7 @@
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>exec-maven-plugin</artifactId>
-                    <version>1.6.0</version>
+                    <version>1.6.1</version>
```
And here 😉

dwijnand commented:
Thanks, @ihostage 😆

ignasi35 commented on grpc-example/grpc-example-java/project/build.properties:
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.8
```
gRPC samples [used to](https://github.com/lagom/lagom-samples/pull/104) have some issue with sbt `1.3.x`. I hope it's fixed now, but we may need to bump `akka-grpc`/`play-grpc` too (or stay on `1.2.8`).

dwijnand committed:
Upgrade to ScalaTest 3.1.1

Something through Lagom 1.6.1 uses ScalaTest 3.1, which is binary incompatbile with 3.0, so we need to upgrade.
